pkg/cloudclient/gcp/private.go

```diff
@@ -0,0 +1,376 @@
+package gcp
+
+//Features to add - image-id, kms-key-id
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"regexp"
+	"time"
+
+	"golang.org/x/oauth2/google"
+	computev1 "google.golang.org/api/compute/v1"
+
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	handledErrors "github.com/openshift/osd-network-verifier/pkg/errors"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type createComputeServiceInstanceInput struct {
+	ContOptImageID string
+	vpcSubnetID    string
+	userdata       string
+	zone           string
+	machineType    string
+	instanceName   string
+	sourceImage    string
+	networkName    string
+}
+
+var (
+	// TODO find a location for future docker images
+	networkValidatorImage string = "quay.io/app-sre/osd-network-verifier:v0.1.159-9a6e0eb"
```

Comment:
TODO: (please open a card) having two different variables for aws and gcp docker image will likely cause some issues in the future. This variable should be moved to some comment config file.

Reply:
Created https://issues.redhat.com/browse/OSD-12769